init.sh

```diff
@@ -125,6 +138,13 @@ then
     cd ${BASE_DIR}
     echo "DONE"
 
+    if [[ "${TARBALL^^}" != "" ]]
+    then
+        RESTORE_DIR=${TARBALL}
+        rm -rf ${RESTORE_DIR}
+        su -pc "tar -xvf ${TARBALL}.gz ${RESTORE_DIR}" ${USER}
```

Comment:
Same here, we should think about `exec` and pipes or `&&`.  
However, it's more important for the `mydumper` / `myloader` commands as they might run for a longer time than the `tar` command.